Schema_refinery/schema_refinery.py

```diff
@@ -0,0 +1,121 @@
+"""Main module."""
```

Comment:
Module docstring should follow the format used in other modules based on Numpydoc ( see [ncbi_linked_ids.py](https://github.com/B-UMMI/Schema_Refinery/blob/6a22b12172b683eb8e99becf3f235ceb1e4b619c/Schema_refinery/Download_module/ncbi_linked_ids.py) ).